### PR TITLE
W-14153240-CompositeSKU-ESB

### PR DESCRIPTION
**Pull request by emileeburkeVT**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: en/mulesoft/modules/ROOT/pages/ms_composer_hyperautomation.adoc [hunk 1]
```diff
@@ -0,0 +1,31 @@
+= MuleSoft Automation Package 
+
+The https://www.mulesoft.com/automation-pricing-new[MuleSoft Automation Package] enables you to log in to Anypoint Platform to access MuleSoft Composer and xref:rpa-home::index.adoc[MuleSoft Robotic Process Automation (RPA)]. With the Hyperautomation Package, MuleSoft Composer now has everything you need to connect apps and create automations.
```
Comment (Cristian-Venticinque) [inline en/mulesoft/modules/ROOT/pages/ms_composer_hyperautomation.adoc:3]: By reading this article, it is not clear to me if the Automation and Hyperautomation are two different Packages with different features, or just the same thing with different names.

File: en/mulesoft/modules/ROOT/pages/ms_composer_hyperautomation.adoc [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+=== Admin: Grant Access to Composer Users 
+
+To access Composer via Anypoint Platform, the Anypoint Admin must first: 
+
+. Create an xref:access-management::managing-your-account.adoc[Anypoint Platform Account] for the users.
+. Grant one of the following xref:ms_composer_prerequisites.adoc#change-user-roles[roles] to each user: 
+.. *Composer Base User*: This user can create and manage their own flows. 
+.. *Composer Super User*: This user can create and manage their own flows. Additionally, this user can take ownership and manage any flows in the Anypoint Composer org. The xref:ms_composer_automation_credits.adoc[Usage Summary] is  only available to those with the Composer Super User role.
+
+=== Access Composer with the Hyperautomation Package
```
Comment (Cristian-Venticinque) [inline en/mulesoft/modules/ROOT/pages/ms_composer_hyperautomation.adoc:24]: Is is really necessary to add "with the Hyperautomation Package" are this instructions different from just the usual way of accessing Composer? 

If that's the Case, I'd suggest a rewording of the titles from this:   

== Access Composer with Hyperautomation Package 
...
=== Admin: Grant Access to Composer Users 
...
=== Access Composer with the Hyperautomation Package  
...

To something like: 


== Access Composer with Hyperautomation Package 
(Add some intro text, like:)
To Access Composer with the Hyperautomation package, first grant access to users, and then log in to Anypoint Platform and open Composer. 
=== Admin: Grant Access to Composer Users 
...
=== Access Composer from Anypoint Platform
...

**Review by Cristian-Venticinque — DISMISSED** [review]
Please check the comments before merging

File: en/mulesoft/modules/ROOT/pages/ms_composer_hyperautomation.adoc (hunk 1 above)
Comment (emileeburkeVT) [inline en/mulesoft/modules/ROOT/pages/ms_composer_hyperautomation.adoc:3]:
```suggestion
The https://www.mulesoft.com/automation-pricing-new[MuleSoft Automation Package] enables you to log in to Anypoint Platform to access MuleSoft Composer and xref:rpa-home::index.adoc[MuleSoft Robotic Process Automation (RPA)]. With the MuleSoft Automation Package, MuleSoft Composer now has everything you need to connect apps and create automations.  
```

**Review by Cristian-Venticinque — DISMISSED** [review]

File: en/mulesoft/modules/ROOT/pages/ms_composer_hyperautomation.adoc [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+With the MuleSoft Automation Package, the following Anypoint Platform features are available to Composer users: 
+
+* xref:access-management::audit-logging.adoc[Anypoint Audit Logging]
+* xref:exchange::index.adoc[Anypoint Exchange] 
+* xref:visualizer::index.adoc[Anypoint Visualizer] 
+* xref:monitoring::index.adoc[Anypoint Monitoring] 
+
+== Access Composer with MuleSoft Automation Package 
+
+To Access Composer with the MuleSoft Automation package, first grant access to users and then log in to Anypoint Platform and open Composer.
```
Comment (sathya0) [inline en/mulesoft/modules/ROOT/pages/ms_composer_hyperautomation.adoc:15]:
```suggestion
To access Composer with the MuleSoft Automation package, first grant access to users and then log in to Anypoint Platform and open Composer.
```

**Review by sathya0 — APPROVED** [review]

**Review by valkyrie69 — APPROVED** [review]

**Pull request merged**